themes/app/src/scss/_variables.scss

```diff
@@ -2,6 +2,8 @@
  * Application variables
  */
 
+$contentPadding: 20px 40px;
```

Comment:
I see, good idea on keeping this consistent across all styles 👍 

Everything else looks good, good job :)